src/platform/AMBD/AMBDConfig.cpp

```diff
@@ -0,0 +1,332 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019-2020 Google LLC.
+ *    Copyright (c) 2018 Nest Labs, Inc.
```

Comment:
Is this a new file or does it indeed need to keep the copyrights from nest and google?

Reply:
It actually doesn't need the additional copyright. I was taking other platforms as reference.
Thanks for point it out, I will remove it.